exampleSite/content/encounters/uwu.md

```diff
@@ -0,0 +1,15 @@
+---
+title: The Weapon's Refrain
+card_image: /theme-assets/gaia-test.png
+tier_image: /theme-assets/gaia-test.png
+encounter_category: ultimate
+tier_name: Ultimate
+series_name: Ultimate
+weight: 2
+tier_weight: 1
+coming_soon: false
+spoiler: false
+expansion: Stormblood
```

Comment:
Maybe this is a personal stylistic nit, but to save us from typing, we could probably abbreviate everything as `arr`, `hw`, `sb`, `shb` and `ew` and just expand when we hit frontend things.